layouts/partials/home/ide.html

```diff
@@ -1,9 +1,9 @@
 <div id="carousel-ide" class="text-left carousel-ide">
     <div class="bg-gray-400 rounded-t border-r-2 border-gray-500">
         <div class="flex p-2">
-            <div class="bg-red-600 rounded-full h-3 w-3"></div>
-            <div class="bg-yellow-600 rounded-full h-3 w-3 mx-2"></div>
-            <div class="bg-green-600 rounded-full h-3 w-3"></div>
+            <div class="rounded-full h-3 w-3" style="background-color: #ff605c;"></div>
```

Comment:
We should update this image to the same as the Resource Search insights page image please.